packages/grafana-ui/src/components/Chart/TooltipContainer.tsx

```diff
@@ -57,7 +57,7 @@ export const TooltipContainer: React.FC<TooltipContainerProps> = ({ position, of
       x: position.x + offset.x - xO,
       y: position.y + offset.y - yO,
     });
-  }, [tooltipRef, position]);
+  }, [tooltipRef, position, offset.x, offset.y, width, height]);
```

Comment:
nit: Best practices is to destruct nested props and use them in the dependencies.

Reply:
done